api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/TestConstants.java

```diff
@@ -26,5 +29,6 @@
     public static final String TEST_EXCEPTION_MESSAGE = "test exception";
     public static final RuntimeException TEST_EXCEPTION = new RuntimeException(TEST_EXCEPTION_MESSAGE);
     public static final LocalDate OVERRIDE_UNTIL_FIVE_YEARS = LocalDate.now().plusYears(5);
+    public static final List<LocalDate> CHILD_BORN_IN_FUTURE = singletonList(LocalDate.now().plusDays(1));
```

Comment:
Could we call this `CHILD_DOB_IN_FUTURE` please as being born is the future is somewhat impossible. :-)

Reply:
It's only used to validate that the future DOB is rejected - so I think it being impossible is valid :)